infrastructure/conf/mock-auth-server.json

```diff
@@ -0,0 +1,25 @@
+{
+  "interactiveLogin": true,
+  "tokenCallbacks": [
+    {
+      "issuerId": "oxauth",
+      "tokenExpiry": 120,
+      "requestMappings": [
```

Comment:
Can this mock issue access tokens and refresh tokens?  Out of scope for here I think but it would be worth putting on the board if possible since we definitely use that with SiC.

Reply:
yep, i believe that part of the process is already happening automatically behind the scenes, at least it seems to be happening from the auth server logs :)

as i half-ass rambled above, my understanding is this: whenever the access token expires, it fetches a new access token from the auth server using the refresh token, all according to the oauth2 dance